### ISSUE TITLE
Transparent  status bar

### ISSUE THREAD

**Issue by ant-ia** [daimajia/AndroidImageSlider#292]
![screenshot_2017-01-27-15-24-43](https://cloud.githubusercontent.com/assets/4062455/22480469/c90b16e4-e7f1-11e6-98c8-53f94761eb19.png)

Hi,
I can not make transparent the status bar
This is the code:
`            <com.daimajia.slider.library.SliderLayout
                android:id="@+id/slider"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:fitsSystemWindows="true"
                app:layout_collapseMode="parallax"
                android:scaleType="centerCrop"
                />`

But with a simple ImageView works perfectly.

Suggestions? Thank you

**Comment by UsmanGhauri** [comment]
its not library issue. you need to setup a scrolling activity. if you're using android studio create a new activity from activity gallery presets
![screenshot_4](https://cloud.githubusercontent.com/assets/10024551/23092384/ca4f5078-f5da-11e6-8be0-2c938df2cc10.png)

put this slider layout inside of that header and keep the header non scroll-able aka fixed and done you're good to go

**Comment by ant-ia** [comment]
Hi @UsmanGhauri 
Thank you for the reply. 
Already use a com-scrolling layout and say so.
But still I can not make the status bar transparent.
This is the layout of the code:  http://pastebin.com/zmqVqfG2

**Comment by UsmanGhauri** [comment]
Did you set your status bar to be transparent? if so then it shows properly. see the attached image i did as sample 
![screenshot_20170219-124012](https://cloud.githubusercontent.com/assets/10024551/23101286/136f9d32-f6a1-11e6-8b25-634a76a64e64.png)

**Comment by ant-ia** [comment]
Hi @UsmanGhauri ,
Sorry but not work for me. 


i have set the status bar to be transparent.
I can not have the effect as the picture that you posted before you.
If I replace your component with a ImageView, it works.

**Comment by UsmanGhauri** [comment]
@ant-ia is it still scroll able? try scrolling up and see if its showing transparent status bar. in my case, if i stretch the header appbar layout to max height it shows status bar with colorprimary but when i scroll up, it shows transparent status bar untill it has no more space and inflates into toolbar with colorprimary

**Comment by ant-ia** [comment]
hi @UsmanGhauri,
I want to have pretty much the opposite. ie:
t0: transparent status bar (with image slider below)
t1: when scrolling up, the status bar and toolbar take on the coloration colorPrimary

This is an example of how I would like to behave. (Sorry if I repeat it, but with the layout I posted above and will work with ImageView)

- [sample-demo](https://camo.githubusercontent.com/d3e779a28f262eeac9fa31b351a751d518ab72a2/687474703a2f2f7265732e636c6f7564696e6172792e636f6d2f64786e616f6b7673662f696d6167652f75706c6f61642f635f7363616c652c775f3235322f76313435303132303233332f657a6769662e636f6d2d6f7074696d697a655f74736e7961662e676966)

**Comment by UsmanGhauri** [comment]
@ant-ia : yes it acts the same way as you mentioned in the that sample-demo link. Only that 'status bar' not the toolbar, is colored when fully stretched as defined in the xml file. and when you scroll up both status bar and toolbar are colored to colorprimarydark and colorprimary

**Comment by ant-ia** [comment]
@UsmanGhauri 
true, as you say above, it works correctly during scrolling.
But I can not get the status bar transparent in the initial state, when it's all extended, or I can not have this:
[how I want](https://cloud.githubusercontent.com/assets/10024551/23101286/136f9d32-f6a1-11e6-8b25-634a76a64e64.png)

but I can get only so:
[as is](https://cloud.githubusercontent.com/assets/4062455/22480469/c90b16e4-e7f1-11e6-98c8-53f94761eb19.png)

**Comment by smilecs** [comment]
This problem may be coming from your overall app theme. check your app theme should be extending an appcompat theme that has no actionbar so the library can set toolbar theme correctly

**Comment by ant-ia** [comment]
hi @UsmanGhauri  , @smilecs 
Trying to use a theme no action bar, as you say, still does not work

theme: 

```

    <style name="Theme.AppCompat.Light.NoActionBar" parent="@style/Theme.AppCompat.Light">
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
        <item name="windowNoTitle">true</item>
        <item name="windowActionBar">false</item>
        <item name="android:windowFullscreen">false</item>
        <item name="android:windowTranslucentStatus">true</item>
        <item name="android:windowContentOverlay">@null</item>
    </style>

```
